cmd2.py

```diff
@@ -500,6 +518,7 @@ class Cmd(cmd.Cmd):
     excludeFromHistory = '''run ru r history histor histo hist his hi h edit edi ed e eof eo eos'''.split()
     exclude_from_help = ['do_eof', 'do_eos']  # Commands to exclude from the help menu
     reserved_words = []
+    use_argument_list = False
```

Comment:
This class member shouldn't be added.  It creates massive backwards compatibility problems with ``cmd``.